src/graphql/types/object/btc-wallet.ts

```diff
@@ -15,7 +15,7 @@ const BTCWallet = new GT.Object({
   fields: () => ({
     id: {
       type: GT.NonNullID,
-      resolve: (source) => source.publicId,
+      resolve: (source) => source.walletId,
```

Comment:
On local testing I got source as type `Wallet`. Is that right or does something else have to change as well?
```suggestion
      resolve: (source: Wallet) => source.id,
```

Reply:
fix in last commit